admin_manual/maintenance/migrating.rst

```diff
@@ -107,7 +107,7 @@ Import on new server::
 
       rsync -Aaxt config data root@new_server_address:/var/www/owncloud 
 
-.. warning:: You must keep the data/ directory’s original filepath. Do not change this!
+.. warning:: If you want to move your datadirectory to another location on the target server, it is advised to do this as a second step. Please see the datadirectory migration [document](https://doc.owncloud.com/server/10.0/admin_manual/maintenance/manually-moving-data-folders.html) for more details.
```

Comment:
Such direct links are getting outdated very soon. A better way to link to a specific part of the documentation is the following:

1. Add ``.. _datadir_move_label:`` below the ``=====================================`` headline in https://github.com/owncloud/documentation/blob/master/admin_manual/maintenance/manually-moving-data-folders.rst
2. Use:

 ```
:ref:`datadir_move_label`
```

 for this link here (the link name will then have the name "How To Manually Move a Data Directory")